knexfile.ts

```diff
@@ -5,9 +5,9 @@ module.exports = {
     client: "postgresql",
     connection: {
       host: "localhost",
-      database: "hts",
-      user: "hu-tao-mains",
-      password: "hu-tao-mains",
+      database: "postgres",
```

Comment:
Will this break me, boom, and five's database settings?

Reply:
oh no